CHANGELOG.md

```diff
@@ -22,6 +22,10 @@ x.x.x Release notes (yyyy-MM-dd)
   service indicates that their credential is no longer valid.
 * Address high CPU usage and hangs in certain cases when processing collection
   notifications in highly-connected object graphs.
+* The keychain service name used by Realm to manage the encryption keys for
```

Comment:
"keychain item name" seems like more correct terminology here, and elsewhere in this entry.